src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -330,7 +330,7 @@ void ConfigureLogger(ParseResult parseResult)
 
     logger = logManager.GetClassLogger();
 
-    string logLevel = parseResult.GetValue(BasicOptions.LogLevel) ?? "info";
+    string? logLevel = parseResult.GetValue(BasicOptions.LogLevel);
```

Comment:
Why's that?

Reply:
Otherwise, nlog configuration file always got overridden.